README.md

```diff
@@ -43,41 +46,59 @@ jobs:
 
 ### I got a website, how do I make it better?
 
-We believe in maximum input that can be infered directly from source. Hence to enable you in building docs, we need no configuration. The docusaurus theme GitHub action infers information from your repository with sensible defaults. Here's some of the things the builder looks out for: 
+We believe in maximum input that can be infered directly from source.
+Hence to enable you in building docs, we need no configuration.
+The docusaurus theme GitHub action infers information from your repository with sensible defaults.
+Here's some of the things the builder looks out for:
 
-**Pre-requistes for running the GitHub action on Flowzone**
+#### Pre-requistes for running the GitHub action on Flowzone
 
 1. A README file in the root of the repository
-2. An input named as `cloudflare_website` in the flowzone.yml file. 
+2. An input named as `cloudflare_website` in the flowzone.yml file.
 
-**Good to have**
+#### Good to have
 
 1. Markdown docs present in either a docs directory called `docs` or `balena-docs`
-2. [Assets/images/files](https://docusaurus.io/docs/markdown-features/assets) being used in the docs being present inside the docs directory as mentioned in point 1 referenced using [markdown links](https://docusaurus.io/docs/markdown-features/links). 
+2. [Assets/images/files](https://docusaurus.io/docs/markdown-features/assets) being used in the docs being present inside the docs directory as mentioned in point 1 referenced using [markdown links](https://docusaurus.io/docs/markdown-features/links).
 3. A favicon file present in the root named as `favicon.ico`
 4. A logo file present in the root named as `logo.png`
 
-This list is exhaustive. Will contine to add to it. Docs are constantly evolving, but always degrading. A little attention to detail will be neeed. 
+This list is exhaustive. Will contine to add to it.
+Docs are constantly evolving, but always degrading.
+A little attention to detail will be neeed.
 
-## Testing sites locally 
+## Developing your site locally
 
-Atm, we don't support local builds.
-To test your content on docusarus conventions, checkout https://docusaurus.io/docs/playground
-To preview how your website will end up looking, make your PR on a docusaurus-enabled repository and the preview URL will be generated. 
-To add new features to the theme, you can start contributing here!
+To develop your site locally you should to have the following installed:
 
+- [`Docker`](https://docs.docker.com/get-docker/)
+
+With `Docker` installed, open a shell in your terminal, navigate to the root of your project's repo and run the following command:
+
+```bash
+docker run -t -i -e "DEV=true" -v $(pwd):/app/local -p 80:80 product:latest
+```
+
+> Note - if you are trying to run this from Windows, please use powershell
```

Comment:
I would add a "Why" to this statement to let the user know why you need recommend that